tests/e2e/support/commands.js

```diff
@@ -70,3 +71,42 @@ Cypress.Commands.add('createOrder', (cartDraft, orderDraft) => cy.wrap(clientPro
         const draft = Object.assign({}, orderDraft, { id: cart.id, version: cart.version });
         return mutation.createOrder(client, draft);
       })))));
+
+
+Cypress.Commands.add('deleteOrder', ({ orderNumber }) => {
```

Comment:
Why is this `deleteOrder` command here? I couldn't find where you are using it.

Reply:
Alex worked on this file @plaskevich